Instruments/microwave_tuning_gui.py

```diff
@@ -134,18 +135,18 @@ def generate_data(self):
                 self.slider_min.value(),
                 "slider max",
                 self.slider_max.value())
-        if len(self.line_data) > 6:
-            self.line_data.pop(0)
-            self.x.pop(0)
```

Comment:
moved this down to regenerate plots since this pertains to the plotting not so much the generation of data